tools/istio-clean-iptables/pkg/cmd/cleanup.go

```diff
@@ -21,17 +21,17 @@ import (
 
 func flushAndDeleteChains(ext dep.Dependencies, cmd string, table string, chains []string) {
 	for _, chain := range chains {
-		ext.RunQuietlyAndIgnore(cmd, "-t", table, "-F", chain)
-		ext.RunQuietlyAndIgnore(cmd, "-t", table, "-X", chain)
+		ext.RunQuietlyAndIgnore(cmd, "-w", "-t", table, "-F", chain)
```

Comment:
For consistency, it would be good to add the timeout to the commands in this file as well.
Not a blocker, though.